.dockerignore

```diff
@@ -1,5 +1,5 @@
 # dockerignore file to minimalize the size of docker image
 #
 # Ignore .git and dev directories
-.git
-dev
+/**
```

Comment:
What will be ignored??

Reply:
Everything in the repo directory when using automated builds or a build locally. Else the docker context scales exponentially. As we are cloning the repo inside the Docker container, this would ensure the whole repo directory is not copied into the image.